www_src/pages/make/make.jsx

```diff
@@ -24,15 +26,14 @@ var Make = React.createClass({
     this.load();
   },
   onError: function (err) {
-    console.error(err);
+    reportError("Error loading project", err);
     this.setState({loading: false});
   },
   onEmpty: function () {
-    console.log('No projects found');
+    reportError('No projects found');
```

Comment:
We should not show an error when no projects exist as it makes for a poor first-time-use.